website/content/docs/plugins/creation/index.mdx

```diff
@@ -247,16 +247,31 @@ scope of a plugin.
 
 ### Testing Plugins
 
-Making your unpublished plugin available to Packer is possible by either:
+To install plugins from local source, you can use the `packer plugins install` command with the `--path` flag:
 
-- Starting Packer from the directory where the plugin binary is located.
-- Putting the plugin binary in the same directory as Packer.
+```shell-session
+$ packer plugins install --path <path-to-binary> <hostname>/<namespace>/<plugin-name>
+```
+
+For example lets install happycloud plugin from a locally sourced binary
+
+```shell-session
+$ packer plugins install --path packer-plugin-happycloud github.com/hashicorp/happycloud
+```
+This will install the happycloud plugin from the `packer-plugin-happycloud` binary so Packer can discover it.
```

Comment:
Nit: one extra empty line here for aeration purposes

```suggestion

This will install the happycloud plugin from the `packer-plugin-happycloud` binary so Packer can discover it.
```